src/pentago.py

```diff
@@ -1,32 +1,45 @@
 import sys
 
-from board import construct_board, add_marble_to_board, is_board_full, get_position_if_valid
+from board import construct_board, add_marble_to_board, is_board_full, get_position_if_valid, rotate_quarter_of_board
 from render import print_board
 
 def init_game():
     try:
         board = construct_board()
 
         while is_board_full(board) == False:
-            board = play_turn(board)
+            board = place_marble(board)
```

Comment:
Not fan of the name. This code is not "playing", it's asking the next player to add a marble.

I know it's not easy to find a method name for that, but `play_turn` is both misleading and too vague for me.

Same for the next line.